Lib/http/server.py

```diff
@@ -909,6 +951,30 @@ def guess_type(self, path):
             return guess
         return 'application/octet-stream'
 
+    def parse_range(self):
+        """Return a tuple of (start, end) representing the range header in
+        the HTTP request. If the range header is missing or not resolvable,
+        None is returned. This only supports single part ranges.
```

Comment:
Let's also add in the NEWS/docs that it's for *single* part ranges (and thus it's probably better to find a link to some RFC content if we can)